core/whistle_apps-1.0.0/src/whapps_controller.erl

```diff
@@ -59,13 +59,12 @@ start_app(App) ->
 stop_app(App) when not is_atom(App) ->
     stop_app(wh_util:to_atom(App));
 stop_app(App) ->
-    _ = application:stop(App),
-    lager:info("stopped kazoo application ~s", [App]).
+    case application:stop(App) of
+      ok -> lager:info("stopped kazoo application ~s", [App]), ok;
```

Comment:
lager's log levels all return 'ok', so no need to have the explicit ok afterwards.

I'd say to log the error tuple as well since these functions are typically called via the SUP and not in the shell.

I'd also consider making these io:format/2 for that reason too...thoughts?